finals-2024-01-16/code/5040-dara-poposka/src/app/country-detail-component/country-detail-component.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { CountryService } from '../services/country.service';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-country-details',
+})
+export class CountryDetailsComponent {
+  country = {};
+
+  constructor(private countryService: CountryService, private route: ActivatedRoute) {}
```

Comment:
Assuming that there is a `CountryService` providing the required data, +5 points for injecting and using `ActivatedRoute`